mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/Uid.test

```diff
@@ -32,6 +32,12 @@ S: \* 2 FETCH \(UID 3\)
 S: \* 3 FETCH \(UID 4\)
 S: a OK FETCH completed.
 
+C: a2 FETCH 1:* (UID)
```

Comment:
In a file name `UID` I would expect tests about `UID ...` not about Fetch used in MSN mode.

Those tests are valuable, maybe they can be move in an existing `fetch...` script?